src/signals/IncidentMap/components/FilterPanel/FilterCategoryWithSub.test.tsx

```diff
@@ -75,9 +75,16 @@ describe('FilterCategoryWithSub', () => {
 
   it('should hit the subcategory toggle', function () {
     renderFilterCategoryWithSub()
+
+    const chevron = screen.getByRole('button', {
+      name: 'Toon meer filter opties',
+    })
```

Comment:
before clicking the chevron you can make sure that the checkbox is not in the document
by: 

    const checkBox1 = screen.queryByRole('checkbox', {
      name: /mockSubCategory_display1/,
    })
    
    expect(checkBox1).toBeInTheDocument()

Reply:
otherwise we're guessing that the checkbox wasn't rendered